app/utils/pusher.js

```diff
@@ -28,6 +28,15 @@ TravisPusher.prototype.init = function (config, ajaxService) {
   if (config.host) {
     Pusher.host = config.host;
   }
+
+  if (ENV.pusher.debug) {
+    Pusher.log = function (message) {
+      if (window.console && window.console.log) {
```

Comment:
Appears to be the case with some IE versions at least, so probs safer this way: https://stackoverflow.com/a/5473193